specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -257,6 +257,82 @@
                     }
                 }
             }
+        },
+        "/certs": {
+            "get": {
+                "tags": [
+                    "attestation"
+                ],
+                "summary": "Retrieves the OpenID Configuration data for the Azure Attestation Service",
+                "description": "Retrieves attestation signing keys in use by the attestation service",
+                "operationId": "Certs_Get",
+                "consumes": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "type": "object"
+                        }
+                    },
+                    "400": {
+                        "description": "Error processing the request",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Reset Policy": {         
+                      "$ref": "./examples/Get_Certs.json"         
+                    }
+                }
+            }
+        },
+        "/.well-known/openid-configuration": {
+            "get": {
+                "tags": [
+                    "attestation"
+                ],
+                "summary": "Retrieves the OpenID Configuration data for the Azure Attestation Service",
+                "description": "Retrieves metadata about the attestation signing keys in use by the  attestation service",
+                "operationId": "MetadataConfiguration_Get",
```

Comment:
This is an unusual naming convention, could you share the reference for this particular naming convention? I would have expected something more like "GetMetadataConfiguration".

Reply:
operationId naming convention: Resource_Action
required by the reviewer from the 1st pull request: https://github.com/Azure/azure-rest-api-specs/pull/7521

Reply:
If that is the standard for azure APIs, then please update the AAS swagger documents to match. I would like to know where the standard for this is - I didn't see it in the Azure API guidelines. Please ask the original reviewer for a reference to that API naming convention since it is counter to the naming conventions for most languages.

We absolutely MUST keep the swagger documents used to build AAS and the swagger documents used for the SDK in 100% byte-for-byte sync, so any changes made to the SDK need to be reflected back in AAS.